### ISSUE TITLE
Batch compressed images

### ISSUE THREAD

**Issue by Eyion** [matiasbenedetto/squoosh-desktop-app#16]
This is a great project, but it can only compress one picture at the same time, which doesn't seem to meet people's demand for batch compression. I hope you will add this function, thank you!

**Comment by zzy-life** [comment]
This is a great project, but it can only compress one picture at the same time, which doesn't seem to meet people's demand for batch compression. I hope you will add this function, thank you!

**Comment by LucasRTonin** [comment]
I agree it would be really useful if we could do batch compression, maybe even one image after the other

**Comment by 3xploiton3** [comment]
+1 please support batch bulk compress

**Comment by danielrdotcom** [comment]
This is a big one, batch support would make this the de-facto for web compression

**Comment by gabyu** [comment]
That would be great yes. My typical use case is to compress 10+ images with a similar compression setting.
This could be achieved by adding multiple files from a browse button and an option to choose which compression setting to use. To give you an idea of what the 'paid competitor' product provides, using Squoosh would definitely be a winner.

![CleanShot 2024-12-28 at 15 55 44](https://github.com/user-attachments/assets/c33bb460-6158-438e-b866-66e706b588a8)
